src/actions/index.js

```diff
@@ -41,3 +41,24 @@ export function setAppsMetricsError(error: Object): Object {
 		error,
 	};
 }
+
+export function createApp(options:Object):Object {
+	return {
```

Comment:
Please use action generator here (`createAction`). You can import it from `batteries/modules/actions/utils`.